### ISSUE TITLE
Change some game content and appstream description

### ISSUE THREAD

**Issue by bbhtt** [flathub/com.voxdsp.TempleDriver#2]
Hello @BeverlyCode I unarchived the repo. Please make a PR with the following changes:

 - Please change this line to remove references to killing/running people/government agents over https://github.com/mrbid/TempleDriver/blob/3e84159ef9fa07dcbfe891f4d7d673c4dbbbe792/flat/templedriver.appdata.xml#L11C1-L11C112

- Please remove the initial voice in the game that mentions to running CIA over with the car.

In short please remove any references to his controversial comments that might make people uncomfortable.

- Please add a line about his illness in the last `<p>` tag to add more context.

**Comment by bbhtt** [comment]
If you don't agree with the changes, you can make them Flathub only and not upstream.
